### PR TITLE
Check for VcrTest being a selector expression

### PR DESCRIPTION
**Pull request by trodge**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: tools/missing-test-detector/testdata/covered_resource_test.go [hunk 1]
```diff
@@ -5,10 +5,11 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/acctest"
```
Comment (zli82016) [inline tools/missing-test-detector/testdata/covered_resource_test.go:8]: I am so familiar how the tool `missing-test-detector` works. Does it matter to import the TPG or TPGB provider?

File: tools/missing-test-detector/testdata/covered_resource_test.go [hunk 2]
```diff
@@ -5,10 +5,11 @@ import (
...
 )
 
 func TestAccCoveredResource(t *testing.T) {
-	VcrTest(t, resource.TestCase{
+	acctest.VcrTest(t, resource.TestCase{
```
Comment (zli82016) [inline tools/missing-test-detector/testdata/covered_resource_test.go:12]: `VcrTest` is also used in other files in the same folder. Should all of `VcrTest` be replaced with `acctest.VcrTest`?

File: tools/missing-test-detector/testdata/covered_resource_test.go (hunk 1 above)
Comment (trodge) [reply tools/missing-test-detector/testdata/covered_resource_test.go:8]: No, this is just a test file that never actually gets compiled.

File: tools/missing-test-detector/testdata/covered_resource_test.go (hunk 2 above)
Comment (trodge) [reply tools/missing-test-detector/testdata/covered_resource_test.go:12]: Either `VcrTest` or `acctest.VcrTest` will be picked up by the missing test detector after this change.

**Pull request merged**